packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1312,7 +1318,7 @@ class _CupertinoActionSheetActionState extends State<CupertinoActionSheetAction>
         + fontSize * _kActionSheetButtonVerticalPaddingFactor;
 
     return MouseRegion(
-      cursor: kIsWeb ? SystemMouseCursors.click : MouseCursor.defer,
+      cursor: widget.mouseCursor ?? (kIsWeb ? SystemMouseCursors.click : MouseCursor.defer),
```

Comment:
If a CupertinoActionSheetAction is disabled, shouldn't it not become clickable on web?

Reply:
I don't think it can be disabled because onClick is defined as required. So passing null or disabling isn't an option. I tried the same and it didn't work